ci/test_integration_espnet1.sh

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+if [ -z "${GITHUB_PR_LABEL_ESPNET1:-}" ]; then
+    log This PR does not involve ESPnet1 fixes. Skipping...
+    exit 0
+fi
+
```

Comment:
I think the skip check should be executed in github action since we want to use this in local.

Reply:
This was a test of using the labels inside the scripts. I will remove then.

Reply:
I will comment to make as example.